index.js

```diff
@@ -17,24 +17,26 @@ module.exports = function(filteredImports) {
 
       ExpressionStatement: {
         enter: function() {
-          shouldRemove.push(false);
+          if (shouldRemove) {
+            shouldRemove.push(false);
+          }
         },
-        exit: function() {
-          if (shouldRemove.pop()) {
-            this.dangerouslyRemove();
+        exit: function(path) {
+          if (shouldRemove && shouldRemove.pop()) {
```

Comment:
I may be missing a nuance here, but I don't grok why we need these guards now...